.github/scripts/noir-js-test.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -eu
+
+cd /usr/src/noir
```

Comment:
where does `/usr/src` originate from? Is it possible to remove this prefix from build scripts or for needing it to be prefixed everywhere?

Reply:
Thanks for the explanation -- it seems that the CI already has knowledge of `/usr/src/noir` so perhaps only having it in CI actions would be better, like your commit diff does. 

Does doing this make it easier to run locally on a Mac for example without docker?